packages/wdio-utils/src/shim.ts

```diff
@@ -50,6 +50,12 @@ export async function executeHooksWithArgs<T> (this: any, hookName: string, hook
             if (/^(sync|async) skip; aborting execution$/.test(e.message)) {
                 return reject()
             }
+            /**
+            * in case of jasmine, when rejecting, we need to pass the message of rejection as well
+            */
+            if (/^=> marked Pending/.test(e)) {
+                return reject(e)
```

Comment:
I think it is good that you add the error here but I think we should do the same for Mocha to keep them in line

Reply:
i'm not sure if this is suppose to be in mocha, i'm working on Jasmine, so i'm not that familiar with how Mocha